{
    "filename": "5b087972-fee8-4530-bf26-3491cd50b557.csv",
    "title": "string",
    "summary": "Failed to generate summary: cannot import name 'call_gpt_api' from 'app.core.gpt' (/Users/ayush/ai-data-insights-dashboard/backend/app/core/gpt.py)",
    "columns": [
        "PassengerId",
        "Survived",
        "Pclass",
        "Name",
        "Sex",
        "Age",
        "SibSp",
        "Parch",
        "Ticket",
        "Fare",
        "Cabin",
        "Embarked"
    ],
    "size": 62002,
    "created_at": "2025-07-18T01:30:51.466503",
    "hash": "8331e5a2532f5fdb66153a8fc053df08a7dc9d2584f246c7fb1752f0a7139ca5"
}